creating-joy/src/test/java/com/technocrats/creatingjoy/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,143 @@
+package com.technocrats.creatingjoy.repository;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import com.technocrats.creatingjoy.dao.AddressRepository;
+import com.technocrats.creatingjoy.dao.UserRepository;
+import com.technocrats.creatingjoy.entity.Address;
+import com.technocrats.creatingjoy.entity.User;
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@Slf4j
+public class UserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private AddressRepository addressRepository;
```

Comment:
Hi,
As Atif has suggested, do not use other repositories other than the repository you are testing.

Reply:
I've tested whether one to one mapping is working or not for address and user

Reply:
Please remove addressRepository and fetch address using user object itself

Reply:
Done